### DISCUSSION TITLE
My company lost the domain, thus, I cannot access my email address registered to my account !

### DISCUSSION THREAD

**Discussion by ahmed204** [community/community#129497] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

As I mentioned in the header, we have totally lost the domain, and the main github account was registered to an email address on our lost domain. I need to regain access to my account and change the email address to be on another domain.

An help ?

**Comment by ahmad9059** [comment]
Regaining access to your GitHub account when the associated email domain is no longer accessible can be challenging, but there are steps you can follow to attempt recovery. Here's a detailed guide on what you can do:

Step 1: Prepare Your Information
Before you contact GitHub support, gather the following information:

Username: Your GitHub username.
Previous Email: The email address associated with the lost domain.
Current Email: A new email address on a domain you can access.
Proof of Identity: Any proof that you are the owner of the account, such as:
Previous commit logs showing your email.
Proof of identity if you have GitHub Pro or any paid plans (billing information).
Any other relevant information or logs from the account.
Step 2: Contact GitHub Support
Visit GitHub Support:
Go to the [GitHub Support page](https://support.github.com/).

Submit a Support Ticket:

Select the option that best fits your issue. In this case, it will likely be under "Account" or "Login Problems."
Fill out the form with as much detail as possible. Be sure to explain your situation clearly, mentioning that you have lost access to the email domain associated with your account.
Step 3: Verification Process
GitHub Support will likely ask you to verify your identity. This process might include:

Proof of account ownership: Commit logs, contribution history, or any emails you might have from GitHub.
Billing information: If you had any paid plans, providing billing details can help prove ownership.
Step 4: Follow Up
After submitting your request, follow up as needed. Be polite and patient; account recovery can take some time.

Template for Contacting GitHub Support
Here’s a template you can use to contact GitHub support:


Subject: Lost Access to Email Domain - Request for Account Recovery

Dear GitHub Support Team,

I am writing to request assistance in regaining access to my GitHub account. Unfortunately, I have lost access to the email domain associated with my GitHub account. Below are the details of my account:

- GitHub Username: [Your Username]
- Email Address on Lost Domain: [Your Old Email]

I can no longer access the email domain to receive any verification emails. I would like to change the email address associated with my GitHub account to [Your New Email Address]. 

To verify my identity, I can provide the following information:
- [Details of recent commits, repository names, billing information, or any other proof of account ownership]

I appreciate your understanding and assistance in this matter. Please let me know if there is any additional information required to verify my identity.

Thank you for your help.

Best regards,
[Your Name]
[Your Contact Information]



Step 5: Additional Steps (if necessary)
If GitHub support cannot help, consider:

Checking backup emails: If you had set up a secondary email or recovery email.
SSH Key Recovery: If you had set up SSH keys, these might help prove your identity.
Two-Factor Authentication: If enabled, any recovery codes might be useful.
By following these steps, you should be able to recover your GitHub account even without access to the original email domain.

**Comment by ebndev** [comment]
Hi @ahmed204,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
